precompile/contracts/testutils/simulated_helpers.go

```diff
@@ -31,6 +33,14 @@ func NewAuth(t *testing.T, key *ecdsa.PrivateKey, chainID *big.Int) *bind.Transa
 // NewBackendWithPrecompile creates a simulated backend with the given precompile enabled
 // at genesis and funds the specified addresses with 1 ETH each.
 func NewBackendWithPrecompile(t *testing.T, precompileCfg precompileconfig.Config, fundedAddrs ...common.Address) *sim.Backend {
+	t.Helper()
+	return NewBackendWithPrecompileAndOptions(t, precompileCfg, fundedAddrs)
+}
+
+// NewBackendWithPrecompileAndOptions creates a simulated backend with the given precompile enabled
+// at genesis and funds the specified addresses with 1 ETH each. Additional options can be passed
+// to configure the backend.
+func NewBackendWithPrecompileAndOptions(t *testing.T, precompileCfg precompileconfig.Config, fundedAddrs []common.Address, opts ...func(*node.Config, *ethconfig.Config)) *sim.Backend {
```

Comment:
I think this should be the only helper as it covers the other.

Reply:
resolved!